### PR TITLE
Feature page object model

### PR DESCRIPTION
**Pull request by DVA09**
Refactor tests to POM

### CODE REVIEW

File: cypress/e2e/sausedemo-test/test.cy.js [hunk 1]
```diff
@@ -1,97 +1,96 @@
...
-  const USER_NAME = "standard_user";
-  const USER_PASSWORD = "secret_sauce";
+import { cartPageElements } from "../../../PageObjects/PageActions/CartPageActions";
+import { completePageElements } from "../../../PageObjects/PageActions/CompletePageActions";
+import { infoPageElements } from "../../../PageObjects/PageActions/InfoPageActions";
+import { loginPageElements } from "../../../PageObjects/PageActions/LoginPageActions";
+import { mainPageElements } from "../../../PageObjects/PageActions/MainPageActions";
+import { overviewPageElements } from "../../../PageObjects/PageActions/OverviewPageActions";
+import { url } from "../../../PageObjects/PageElements/LoginPageElements.json"
+
```
Comment (Alex19777) [inline cypress/e2e/sausedemo-test/test.cy.js]: Rename this file please, name should be informative

Comment (DVA09) [reply cypress/e2e/sausedemo-test/test.cy.js]: Renamed the file

File: cypress/e2e/sausedemo-test/HomePageTests.cy.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    Main_Elements.shoppingCartButton();
+    Cart_Elements.checkoutButton();
+    Info_Elements.firstName();
+    Info_Elements.lastName();
+    Info_Elements.postalCode();
+    Info_Elements.continueButton();
+    Overview_Elements.finishButton();
+    Complete_Elements.title();
+    Complete_Elements.header();
+    Complete_Elements.textComplete();
```
Comment (Alex19777) [inline cypress/e2e/sausedemo-test/HomePageTests.cy.js]: Add validation on the test level. Check all tests

File: cypress/e2e/sausedemo-test/api.cy.js [hunk 3]
```diff
@@ -1,58 +1,27 @@
 /// <reference types='cypress' />
```
Comment (Alex19777) [inline cypress/e2e/sausedemo-test/api.cy.js]: Change structure of all folders

File: PageObjects/PageActions/MainPageActions.js [hunk 4]
```diff
@@ -1,97 +1,149 @@
...
     cy.get("#logout_sidebar_link").contains("Logout");
+  }
+
+  reset() {
     cy.get("#reset_sidebar_link").contains("Reset App State");
-  });
+  }
 
-  it("Filter test", () => {
+  //Filter Tests
```
Comment (Alex19777) [inline PageObjects/PageActions/MainPageActions.js]: Add separate files for comments

File: cypress/e2e/sausedemo-test/HomePageTests.cy.js (hunk 2 above)
Comment (DVA09) [reply cypress/e2e/sausedemo-test/HomePageTests.cy.js]: Added validation

File: cypress/e2e/sausedemo-test/api.cy.js (hunk 3 above)
Comment (DVA09) [reply cypress/e2e/sausedemo-test/api.cy.js]: Move folder for API tests

File: PageObjects/PageActions/MainPageActions.js (hunk 4 above)
Comment (DVA09) [reply PageObjects/PageActions/MainPageActions.js]: Split files

File: cypress/e2e/Swagger-API/PetstoreAPI.cy.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+/// <reference types='cypress' />
```
Comment (Alex19777) [inline cypress/e2e/Swagger-API/PetstoreAPI.cy.js:1]: Are you sure that this test file is a part of e2e?

Comment (DVA09) [reply cypress/e2e/Swagger-API/PetstoreAPI.cy.js:1]: All tests must be stored in folder 2, otherwise they will not run in Cypress

**Review by Alex19777 — APPROVED** [review]

**Pull request merged**